src/components/auth/Login.js

```diff
@@ -1,13 +1,140 @@
 import React from 'react'
 
-const Login = () =>  {
+
+class LoginForm extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
```

Comment:
Does this component handle its own state?

Reply:
Yes. It handles only form input state. Would you prefer that I also handle that in global state?

Reply:
I think its fine there.

Reply:
I think since we are using redux, let redux handle all the state.

Reply:
So, I should create actions that are dispatched every time a user adds a character in the input fields?